src/app.ts

```diff
@@ -45,6 +47,13 @@ export async function bootstrap({ port }: BootstrapOptions) {
     bodyLimit: constants.FASTIFY_BODY_LIMIT
   }).withTypeProvider<JsonSchemaToTsProvider>();
 
+  // Custom parser for "application/x-protobuf"
+  app.addContentTypeParser(
+    'application/x-protobuf',
+    { parseAs: 'buffer' },
+    traceProtobufBufferParser
+  );
```

Comment:
It's weird that `traceProfobufBufferParser` parses anything protobuf.

Reply:
I need to define the custom `contentTypeParser`, otherwise I get following:
```
OTLPExporterError: Unsupported Media Type
    at IncomingMessage.
```
error. 
The `traceProtobufBufferParser` filters the parser logic only for the `v1/traces` route.

Reply:
can you utilize this somehow?

```
As with the other APIs, addContentTypeParser is encapsulated in the scope in which it is declared. This means that if you declare it in the root scope it will be available everywhere, while if you declare it inside a plugin it will be available only in that scope and its children.
```